### DISCUSSION TITLE
Should I apply a generic CSS to see my components right?

### DISCUSSION THREAD

**Discussion by wsw70** [primefaces/community#1101] [PrimeVue]
I am just starting with PrimeVue and it looks wonderful.

One problem I am having is with the visibility of the various elements vs. the background. As an example, this is what I am playing with:

![image](https://github.com/primefaces/community/assets/1345886/ea6feb1c-bba7-4377-94c9-fa077a36c26c)

My general problem is that there is no contrast between the elements and boundaries are invisible (such as between the `Card` and the `DataTable` that is inside). The radio buttons are practically invisible (I discovered that there were there by chance, I thought that my code was wrong for never showing them).

This corresponds to soma basic code (I removed the other elements visible on the screenshot for clarity):

```vuejs
<Card class="place-items-center z-20">
            <template #title>Configuration</template>
            <template #content>
                <div>
                    <DataTable :value="config.cities" v-model:selection="defaultCity" dataKey="name">
                        <Column field="name" header="City"></Column>
                        <Column field="timezone" header="Timezone"></Column>
                        <Column field="defaultCity" selectionMode="single" header="Default"></Column>
                    </DataTable>
                </div>
            </template>
        </Card>
```

My question: what should I do to make the elements visible, contrasted (somehow) with the background they are on?

**Comment by tugcekucukoglu** [comment]
You could use `pt` property for each component. 

Which theme are you using? Could you provide a simple stackblitz demo?
